src/components/ha-form/ha-form-string.ts

```diff
@@ -12,6 +12,8 @@ import type {
   HaFormStringSchema,
 } from "./ha-form";
 
+const SECRET_FIELDS = ["password", "client_secret", "access_secret", "secret"];
```

Comment:
You don't need `client_secret` and `access_secret` because we check with `includes`, so `secret` matches both.
```suggestion
const SECRET_FIELDS = ["password", "secret"];
```

Reply:
I see, yeah
Should "token" also be checked?

Reply:
not sure. 🤷